packages/bcp/src/cosmwasmcodec.spec.ts

```diff
@@ -1,9 +1,9 @@
-import { Address, PostableBytes, PrehashType, SendTransaction, TokenTicker } from "@iov/bcp";
+import { PostableBytes, PrehashType } from "@iov/bcp";
 import { Encoding } from "@iov/encoding";
 
-import { CosmWasmCodec } from "./cosmwasmcodec";
+import { CosmosCodec } from "./cosmwasmcodec";
```

Comment:
why is the file still named `cosmwasmcodec` if it is now a more generic `CosmosCodec`?

Reply:
God point, I forgot that, will update